src/Symfony/Component/VarDumper/Tests/Caster/ReflectionCasterTest.php

```diff
@@ -208,6 +208,7 @@ public function testGenerator()
     this: Symfony\Component\VarDumper\Tests\Fixtures\GeneratorDemo { …}
     trace: {
       %s%eTests%eFixtures%eGeneratorDemo.php:9 {
+        Symfony\Component\VarDumper\Tests\Fixtures\GeneratorDemo::foo()
```

Comment:
In this case, we are in a trace, so displaying the method here is the wanted behavior.